samples/client/petstore/php/SwaggerClient-php/lib/Model/MixedPropertiesAndAdditionalPropertiesClass.php

```diff
@@ -181,6 +181,11 @@ public function getUuid()
      */
     public function setUuid($uuid)
     {
+        if (is_null($uuid)) {
+            $this->container['uuid'] = $uuid;
+
+            return $this;
+        }
         $this->container['uuid'] = $uuid;
```

Comment:
The problem is in the properties that have validations, for example, setInteger(). If you receive a null and execute
```
if ($integer < 10.0) {
             throw new \InvalidArgumentException('invalid value for $integer when calling FormatTest., must be bigger than or equal to 10.0.');
         }
```
null < 10 => true => The exception is thrown.
The same happens in our API:
```
if (strlen($id_developer) < 1) {
            throw new \InvalidArgumentException('invalid length for $id_developer when calling Program., must be bigger than or equal to 1.');
        }
```
strlen(null) => 0, so the Exception is also thrown.

I understand that for setters without validations, the resulting code makes no sense, like in the example you said.
Maybe a better approach could be to add **if (!is_null($var) && (<any_other_condition>)** in all the validations of the setter in the case the property is not required. This would result in the following setInteger() method:
```
public function setInteger($integer)
      {
          if (!is_null($integer) && ($integer > 100.0)) {
              throw new \InvalidArgumentException('invalid value for $integer when calling FormatTest., must be smaller than or equal to 100.0.');
         }
         if (!is_null($integer) && ($integer < 10.0)) {
             throw new \InvalidArgumentException('invalid value for $integer when calling FormatTest., must be bigger than or equal to 10.0.');
         }
 
         $this->container['integer'] = $integer;
 
         return $this;
     }
```

What do you think @wing328 ?